test/install_test.go

```diff
@@ -151,6 +151,64 @@ func TestCheckPreInstall(t *testing.T) {
 	}
 }
 
+// Gathers success stats from the apps installed in the test-tap namespace
+// Used to ensure upgrade process went smoothly
+func ensureTAppsRunning(t *testing.T) {
+	args := []string{"stat", "deploy", "-n", TestHelper.GetTestNamespace("tap-test"), "-t", "60s"}
+	err := TestHelper.RetryFor(20*time.Second, func() error {
```

Comment:
If I understand this correctly, a 20 second timeout seems super aggressive.  It will take some time or the app to come up and to start processing traffic.  Plus there's up to a 10 second delay from the prometheus scrape interval.

Reply:
@adleong That's very useful context. What would you consider a more suitable value?

Reply:
Not sure. I'd recommend experimenting to see how long it takes for these stats to become available.

Reply:
I see now that this is why my integration test is failing with **No traffic found**